relayer/chains/evm/event_parse.go

```diff
@@ -12,7 +13,9 @@ func (p *Provider) getRelayMessageFromLog(log types.Log) (*providerTypes.Message
 		return nil, fmt.Errorf("topic length mismatch")
 	}
 	topic := log.Topics[0]
-
+	if len(topic) != 0 {
```

Comment:
Wouldn't this logic break the existing implementation as we are doing the bitcoin relay for all the topic messages ?

Reply:
This is just redundant code, we have reviewed but left them out.
I removed them!